Neos.Neos/Classes/Fusion/ConvertUrisImplementation.php

```diff
@@ -61,7 +65,7 @@
 class ConvertUrisImplementation extends AbstractFusionObject
 {
     public const PATTERN_SUPPORTED_URIS
-        = '/(node|asset):\/\/([a-z0-9\-]+|([a-f0-9]){8}-([a-f0-9]){4}-([a-f0-9]){4}-([a-f0-9]){4}-([a-f0-9]){12})/';
+        = '/(node|asset):\/\/(([a-z]*)\/)?([a-z0-9\-]+|([a-f0-9]){8}-([a-f0-9]){4}-([a-f0-9]){4}-([a-f0-9]){4}-([a-f0-9]){12})/';
```

Comment:
The regex should be even broader because node aggregate ids don't have to be uuids (or only consists of a-z characters)